tensorflow-cpu.yaml

```diff
@@ -0,0 +1,126 @@
+#nolint:valid-pipeline-git-checkout-tag
+package:
+  name: tensorflow-cpu
+  version: 0_git20250108
+  epoch: 0
+  description: tensorflow-cpu
+  copyright:
+    - license: Apache-2.0
+  target-architecture:
+    - x86_64
+  dependencies:
+    runtime:
+      - bash
+      - build-base
+      - curl
+      - pkgconf
+      - py${{vars.py-version}}-pip
+      - python-${{vars.py-version}}-dev
+      - unzip
+
+vars:
+  py-version: 3.11
+  pypi-package: tensorflow-cpu
+  prefix: /usr
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - py${{vars.py-version}}-pip
+      - python-${{vars.py-version}}
+  environment:
+    # This is needed to work around the error "ValueError: ZIP does not support timestamps before 1980"
+    SOURCE_DATE_EPOCH: 315532800
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 35a7dbc754193523e5582ce5b5cbbb4c07cff7b2
+      repository: https://github.com/tensorflow/build
+      branch: master
+
+  - working-directory: tensorflow_runtime_dockerfiles
+    pipeline:
+      - name: Install cpu requirements
+        runs: |
+          pip${{vars.py-version}} install \
```

Comment:
Instead of installing directly to site packages, we'll need to install to a virtual environment